src/test/js/shared/idam_helper.js

```diff
@@ -241,6 +241,33 @@ class IdamHelper extends Helper {
          browser.close();
        });
    }
+
+  extractUrl(email) {
+     return (notifyClient
+         .getNotifications("email", "sending")
+         .then(response => {
+             console.log("Searching " + response.body.notifications.length + " emails(s)");
+             var result = response.body.notifications.find(obj => {
+                 if (obj.email_address === email) {
+                     return obj.email_address === email
+                 } else {
+                     console.log("Ignoring unmatched email address");
```

Comment:
Remove the whole else block and replace with a comment that says "NOTE: NEVER LOG EMAIL ADDRESS FROM THE PRODUCTION QUEUE".
Won't be an issue once we create a new notify api key for testing